html/index.html

```diff
@@ -151,7 +151,7 @@ <h1>Settings</h1>
                         <label for="videosCountOnMainPageInput" class="form-label">Videos Count</label>
                         <input type="number" class="form-control is-invalid" id="videosCountOnMainPageInput" name="VideosCountOnMainPage" min="1" max="100">
                         <div class="invalid-feedback">
-                            Please provide a valid number.
+                            <p id="error-text" class="d-none">Please provide a valid number</p>
```

Comment:
id is too generic and can be used for any other element but it should be used only once per page

Reply:
please add error container by javascript and using <template> tag like we did with the list of videos